### PR TITLE
vane-network

### PR DESCRIPTION
**Pull request by MrishoLukamba**
#### Project Abstract

vane-network a polkadot native tokenless parachain that help social-commerce businesses grow, foster trust-less interactions and enabling transaction safety in web3.

vane helps existing social-commerce entrepreneuers to use web3 based services on top of large existing networks such as Polkadot and Ethereum and allowing crypto users to transact DOT, USDT, ETH in a risk free system.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) or BTC address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by MrishoLukamba** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. Regarding the deliveries, could you focus on the specific features that you want to develop that currently don't exist? For example, it's not clear to me how your vane XCM transfer pallet differs from existing solutions. In general, the more technical details you can provide as part of the specification, the better. Additionally, for example, the video link under development status currently points to https://github.com/w3f/Grants-Program/tree/master/applications

**Review by dsm-w3f — CHANGES_REQUESTED** [review]
@MrishoLukamba thank you for the grant application. I agree with David that the application needs more details on the difference between what already exists and what you are proposing. Additionally, I think the calculation of total FTE and duration between the milestones and the total of the grant are not right. The audio of your video demo is very difficult to listen and the pages provided also have broken links. Please double-check this item as well.

**Comment by MrishoLukamba** [comment]
@dsm-w3f @Noc2 , Yes am preparing docs for clarification as the naming are same but everything is implemented different i think I will clarify on that. And thanks for the feedback

**Comment by MrishoLukamba** [comment]
@Noc2 Updated the application

**Comment by MrishoLukamba** [comment]
> @MrishoLukamba thank you for the grant application. I agree with David that the application needs more details on the difference between what already exists and what you are proposing. Additionally, I think the calculation of total FTE and duration between the milestones and the total of the grant are not right. The audio of your video demo is very difficult to listen and the pages provided also have broken links. Please double-check this item as well. 

Updated the changes @dsm-w3f

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks for the application @MrishoLukamba. It looks interesting. However, the formatting of the delivery tables is still off. It should be a list format like in the [template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md). 

Can you adjust it to fix the numbering? If its easier, some of the info can be moved to the application body and then you can link to it.

**Comment by MrishoLukamba** [comment]
> Thanks for the application @MrishoLukamba. It looks interesting. However, the formatting of the delivery tables is still off. It should be a list format like in the [template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md).
> 
> Can you adjust it to fix the numbering? If its easier, some of the info can be moved to the application body and then you can link to it.

Formatted the milestones, hopefully they are readable now. @Noc2 @keeganquigley

**Comment by MrishoLukamba** [comment]
Changes have been addressed
@keeganquigley @Noc2 @dsm-w3f.

**Comment by keeganquigley** [comment]
Thanks for the changes @MrishoLukamba Tanssi is super cool and I'm glad you want to utilize it, but the benefits of this parachain still aren't clear to me. How is your solution more risk-free than existing methods? 

- I understand your frustrations around XCM transfers getting stuck, but these issues are usually compatibility issues that are resolved by the parachain team. The transfers at least stay permissionless. I'm not sure that a "permissioned pallet" is the right solution, because wouldn't this infer that trust in Vane is needed?

**Comment by MrishoLukamba** [comment]
> Thanks for the changes @MrishoLukamba Tanssi is super cool and I'm glad you want to utilize it, but the benefits of this parachain still aren't clear to me. How is your solution more risk-free than existing methods?
> 
> * I understand your frustrations around XCM transfers getting stuck, but these issues are usually compatibility issues that are resolved by the parachain team. The transfers at least stay permissionless. I'm not sure that a "permissioned pallet" is the right solution, because wouldn't this infer that trust in Vane is needed?

Thanks for the queestion @keeganquigley. Vane is not solving XCM infra or improving XCM infra. Vane is a solution which act as an extension safety and product layer on Polkadot and AssetHub for now. What Vane is trying to solve is users can have the ability to transfer their DOT, USDT & USDC in a risk free manner, as small mistakes should not cost crypto users.
As this problem of wrong address transfers and humanely mistakes when doing transactions have hindered lots of new users to try using crypto as there is no a safety net for users.

Apart from that we are trying to build products that utilize trustless interaction such as improving the social-commerce business using blockchain technology. But vane wont be having the native token instead new users and existing users will use these services using their token ( DOT, USDT, USDC). So vane will just act as an extension product layer on existing blockchain network.  

So with all these functionality we will be utilizing XCM. so XCM is a tool for us to enable vane to work as intended. But the tracking of XCM message is to make sure that if anything was to happen we can know how to resolve it based on the latest tracked stage. 

We have already been accepted into tanssi pioneers program so launching an appchain will speed our development to product market fit. As we believe vane will be crucial to improve UX, safety and crypto usability accross web2 social-commerce. 

In the demo I have shown how a user can use DOT to transfer to receivers address in a risk free manner as there is confirmation of receivers address and reversibility of transaction if the sender did anything wrong. And also in one of the milestone we will develop a Notion template marketplace to showcase how individuals can use DOT, USDT to interact with sellers trustlessly using Vane. 

In a nutshell Vane will be the computation of these interaction while the Token will remain to be the existing ones. 
@Noc2 @keeganquigley

**Comment by MrishoLukamba** [comment]
> Thanks for the changes @MrishoLukamba Tanssi is super cool and I'm glad you want to utilize it, but the benefits of this parachain still aren't clear to me. How is your solution more risk-free than existing methods?
> 
> * I understand your frustrations around XCM transfers getting stuck, but these issues are usually compatibility issues that are resolved by the parachain team. The transfers at least stay permissionless. I'm not sure that a "permissioned pallet" is the right solution, because wouldn't this infer that trust in Vane is needed?

And also on "permissioned pallet", This is not implemented on XCM as vane is not XCM solution, vane used XCM to achieve the safety and product layer on existing networks ( Polkadot & AssetHub), the permissionless term in the 3rd milestone is used to limit deployment of contract on vane chain. As vane will be extensible product and the extension will be done using smart contracts in a permissioned way as all products extended should and will be based on using vane core features.

**Review by keeganquigley — APPROVED** [review]
Thanks for all the explanations @MrishoLukamba very informative and has helped me to grasp your vision. This definitely sounds like a common-good solution and happy to see you utilizing Tanssi, the Asset Hub, and XCM so I'm willing to give it a shot. I will mark it as ready for review and ping the rest of the committee.

**Comment by MrishoLukamba** [comment]
Any changes or any clarification request? @Noc2 @dsm-w3f

**Review by semuelle — CHANGES_REQUESTED** [review]
Sorry for the late response, @MrishoLukamba. I have a few questions/requests:

- Please provide a registered address, either in the application or via [email](mailto:grants@web3.foundation).
- Why is your focus on social commerce? What's their incentive/advantage?
- Could you explain the second image? Most importantly, who controls the Multisig and what control do payers and payee have in this setup?

**Comment by MrishoLukamba** [comment]
> Sorry for the late response, @MrishoLukamba. I have a few questions/requests:
> 
> 
> 
> - Please provide a registered address, either in the application or via [email](mailto:grants@web3.foundation).
> 
> - Why is your focus on social commerce? What's their incentive/advantage?
> 
> - Could you explain the second image? Most importantly, who controls the Multisig and what control do payers and payee have in this setup?

Why focus on social commerce? We want to tap into the growing market which can benefit greatly using blockchain abstract features.
- Customisable transactions 
- Order and funds control
- Trust minimisation between customer and online retailer interactions 
- Enabling fast and borderless transactions using stablecoin in assethub

Concerning multisig.
The multisignature as demonstrated in the demo video is for address confirmation and validation between sender and receiver. 

This will solve the issue of wrong address faulty transactions and also enabling reversibility 

And all of this is built on top of polkadot and assethub as users of vane will transfer their dot, usdt, usdc directly using vane without purchasing of any native token( vane) as we are also not planning to have one. As our notion of vane is to be an extension product and safety layer for polkadot and assethub.

**Comment by MrishoLukamba** [comment]
> Thanks for the changes @MrishoLukamba did you also provide a registered physical address? I'm not finding an email for one.

I have included the email addresses and we havent yet registered on a physical address or you meant our physical address ( builders )?

**Comment by keeganquigley** [comment]
Thanks @MrishoLukamba yes I meant the physical address for your team in order to send the payment.

**Comment by MrishoLukamba** [comment]
> Thanks @MrishoLukamba yes I meant the physical address for your team in order to send the payment.



> Thanks @MrishoLukamba yes I meant the physical address for your team in order to send the payment.

We are working remotely and yet to register a physical company address. But I have provided indvidual team member's address. If there is any issues which requires us to have physical company address please let us know. And all payment will be conducted via USDT as the address provided above. @keeganquigley @semuelle @Noc2 @dsm-w3f

**Review by keeganquigley — APPROVED** [review]

**Comment by MrishoLukamba** [comment]
And also am noticing only @keeganquigley replying. How can I get other reviewers to give us feedback? @keeganquigley @Noc2 @semuelle

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Hi @MrishoLukamba since no other committee members have been interested thus far, could you perhaps reduce the scope to remove the 2nd and/or 3rd milestone? This might improve the chances, and you could always apply for follow-up grants at a later stage.

**Comment by MrishoLukamba** [comment]
> Hi @MrishoLukamba since no other committee members have been interested thus far, could you perhaps reduce the scope to remove the 2nd and/or 3rd milestone? This might improve the chances, and you could always apply for follow-up grants at a later stage.

Okey I will remove the first milestone and leave 2nd and 3rd.

But I have 1 question. Arent the committee members supposed to give feedback even if they dont want to vote? As staying quiet is hard to know how they judge the application?

**Review by keeganquigley — DISMISSED** [review]

**Comment by MrishoLukamba** [comment]
We are happy to announce that we have been accepted into Encode Club Polkadot accelerator program @keeganquigley @Noc2 @dsm-w3f

**Comment by semuelle** [comment]
Hi @MrishoLukamba, sorry for the long radio silence. I will provide more feedback in the coming days. 

Also, sorry for the inconvenience, but we [just started requiring KYC/KYB checks](https://grants.web3.foundation/docs/faq#what-is-kyckyb-and-why-do-i-have-to-provide-this-data) for all potential grantees. Could you please provide the information outlined under [this link](https://in.sumsub.com/idensic/l/#/uni_JRqwo72Y5AjXaRDO)? Alternatively, if the beneficiary is not an entity but individual(s) (I can't tell based on the information in the application), please follow [this link](https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7). Let me know if you have any questions or issues!

**Comment by MrishoLukamba** [comment]
> Hi @MrishoLukamba, sorry for the long radio silence. I will provide more feedback in the coming days. 
> 
> 
> 
> Also, sorry for the inconvenience, but we [just started requiring KYC/KYB checks](https://grants.web3.foundation/docs/faq#what-is-kyckyb-and-why-do-i-have-to-provide-this-data) for all potential grantees. Could you please provide the information outlined under [this link](https://in.sumsub.com/idensic/l/#/uni_JRqwo72Y5AjXaRDO)? Alternatively, if the beneficiary is not an entity but individual(s) (I can't tell based on the information in the application), please follow [this link](https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7). Let me know if you have any questions or issues!

Currently we havent registered vane network in any legal boundaries ( country ). 

Will this affect our applications or I can provide my legal documents and tax id?

**Comment by semuelle** [comment]
> Will this affect our applications or I can provide my legal documents and tax id?

All beneficiaries (usually the invoicing entity) have to perform the check, but whether they are an individual or legal entity does not matter. I hope that helps. Feel free to reach out to me (sebastian:web3.foundation on Matrix) if you have any further questions.

**Comment by semuelle** [comment]
Hey @MrishoLukamba, did you receive an email from our KYC provider regarding your application?

**Review by keeganquigley — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]
Sorry for the delay here. Happy to go ahead with it.

**Comment by MrishoLukamba** [comment]
> Thanks for the proposal! Overall looks very interesting, and I think most of the questions I had have been asked before :).
> 
> 
> 
> Team in itself looks well suited for the project, however I wonder how attainable the timelines are with other projects like [Ordum](https://kusama.subsquare.io/referenda/4) which I believe you're also working on and is still going on. Are you confident you'll be able to deliver the project on the proposed timelines?

Thanks for the comment. So at the moment I have parted ways with Ordum.

File: applications/vane-network.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+   - Crypto users for transactions
+   - Social commerce entrepreneurs
+
+**What need(s) does your project meet?**
+  - Introducing safety for web3 mass adoption
+  - Bridging and using web3 based technology to grow online businesses.
+
+**Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?**
+  
+  - No, but ENS and Naing services try to solve the issue of ensuring wrong address mistakes are resolved. But Vane ensures this product to work just outside the box without purchasing any name , just existig addressbut introducing confirmation and reverting system ontop of existing networks.
```
Comment (takahser) [inline applications/vane-network.md:118]:
```suggestion
  - No, but ENS and Naing services try to solve the issue of ensuring wrong address mistakes are resolved. But Vane ensures this product to work just outside the box without purchasing any name, just existing address but introducing confirmation and reverting system on top of existing networks.
```

File: applications/vane-network.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Video tutorial | We will publish an workshop on X/Twitter that explains [...] (what was done/achieved as part of the grant and how the product should be used) |
```
Comment (takahser) [inline applications/vane-network.md:262]: Appreciate the video tutorial, however, could you also add an article, at least to milestone 2? See also the [mandatory deliverables](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#milestone-1-example--basic-functionality) in the application template.

File: applications/vane-network.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+| 3. | Notion template marketplace contract| The marketplace will showcase how we can enable transaction customization for social-commerce business and foster growth. It will contain, listing of notion templates, securely limiting access of the notion template, buying of the template, Order placement and tracking.|
+| 4. | Backend (Nest JS) | All chain ( vane, polkadot/rococo, assethub) api interaction implementation, API chart workflow, notion marketplace implementation for storing notion pages, notion users as a POC |
+| 5. | Frontend (Next JS) |  A front end prototype in Figma, A front end interaction for implemented feature hosted in vercel which will be only for wide screen ( no responsiveness ) as later on (not this grant ) we will implement a mobile application |
```
Comment (takahser) [inline applications/vane-network.md:267]: Usually we require wireframes to be delivered as part of the grant proposal. You can use any suitable tool for that (incl. figma).

**Review by takahser — CHANGES_REQUESTED** [review]
@MrishoLukamba I added a few inline-suggestions and -comments. In addition to that, I'd be curious to know if your prototype includes integration with any social network (such as X or meta, as seen in your diagram).

File: applications/vane-network.md (hunk 3 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:267]: The wireframe as product clickable prototype will be submitted

File: applications/vane-network.md (hunk 2 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:258]: We thought it will be better to not include an article and to be more effective on the video for better engagement and get proper feedback from users/community.

Comment (takahser) [reply applications/vane-network.md:258]: Ok I'm fine with that. But could you add a remark to the doc itself, where you explain this?

File: applications/vane-network.md (hunk 3 above)
Comment (takahser) [reply applications/vane-network.md:263]: Just to be clear - we expect the wireframes to be delivered before we approve the proposal. It doesn't necessarily have to be clickable.

**Review by takahser — CHANGES_REQUESTED** [review]
@MrishoLukamba I added follow-up comments to the inline comments, pls have a look.

**Comment by MrishoLukamba** [comment]
Hello @takahser So on wireframes, we did include that after we start the grant program and we are developing it to be clickable. So for just wireframes, is it okey we submit them next week?

**Comment by semuelle** [comment]
> So for just wireframes, is it okey we submit them next week?

Hey @MrishoLukamba. You can add the wireframes to the application now or next week, there is no rush from our side. But as @takahser said, they don't have to be clickable or particularly polished, so don't spend too much time on them. :)

**Comment by MrishoLukamba** [comment]
> > So for just wireframes, is it okey we submit them next week?
> 
> Hey @MrishoLukamba. You can add the wireframes to the application now or next week, there is no rush from our side. But as @takahser said, they don't have to be clickable or particularly polished, so don't spend too much time on them. :)

Okey

**Review by semuelle — CHANGES_REQUESTED** [review]
Hi @MrishoLukamba. Are you still planning to include wireframes? I can see that you have already made significant progress with the project, they are simply a requirement of any grant application involving UI work. As I said, they don't have to be polished. It seems that you would be well underway to being approved then. 

Regarding the ongoing work on the project: it would be great if you could add a paragraph on that in the `Development Status` section of the grant application. It's currently empty now.

**Comment by MrishoLukamba** [comment]
> Hi @MrishoLukamba. Are you still planning to include wireframes? I can see that you have already made significant progress with the project, they are simply a requirement of any grant application involving UI work. As I said, they don't have to be polished. It seems that you would be well underway to being approved then.
> 
> Regarding the ongoing work on the project: it would be great if you could add a paragraph on that in the `Development Status` section of the grant application. It's currently empty now.

Okey yes they are coming around and the section of development, We updated concerning our ongoing work in the grant application.

**Review by keeganquigley — APPROVED** [review]
Thanks for the changes @MrishoLukamba re-approving.

**Comment by MrishoLukamba** [comment]
> Thanks for including the roadmap @MrishoLukamba re-approving.

Updates on our end: 
I know we have been slow to respond, I am currently finishing PBA, but we will be submitting the wireframes soon.
And will be updating the founders list

**Comment by takahser** [comment]
@MrishoLukamba okay, thanks for the note, and no worries about the delay. I know the PBA program can be exhaustive for students, so I suggest to focus on PBA for now. You can add the wireframes after the PBA, feel free to ping us again, once that's done.

**Comment by MrishoLukamba** [comment]
> @MrishoLukamba okay, thanks for the note, and no worries about the delay. I know the PBA program can be exhaustive for students, so I suggest to focus on PBA for now. You can add the wireframes after the PBA, feel free to ping us again, once that's done. Also, I've put the application _on hold_ for the meantime; I'll change it back to _ready for review_ once the wireframes are added. 👍

Added the wireframes for the risk free transaction feature

File: applications/vane-network.md (hunk 3 above)
Comment (takahser) [reply applications/vane-network.md:272]: wireframes have been added, resolving this comment

File: applications/vane-network.md (hunk 2 above)
Comment (takahser) [reply applications/vane-network.md:267]: This has not been addressed yet. Also, there is still some leftover text present that was copied over from the template:

> [...] (what was done/achieved as part of the grant and how the product should be used)

File: applications/vane-network.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Video tutorial | We will publish an workshop on X/Twitter that explains [...] (what was done/achieved as part of the grant and how the product should be used) |
+| 1. | pallet transaction claim resolver | We will create a Substrate module that will handle failed transaction based on XCM delivery issues. Allowing users to claim their incomplete transactions|
```
Comment (takahser) [inline applications/vane-network.md:268]: Could you outline the supported reasons for xcm delivery failure here?

File: applications/vane-network.md [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+| 2. | permissioned pallet contract & chain extensions | A permissioned wasm contract pallet and chain exensions for vane risk free transfer system to be used to extend Vane product services. Account who intend to extend vane product services must apply to be whitelisted to deploy the contract.<br><br>**Chain extensions**<br> This will be limited to vane specific pallets extrinsics only, which includes, pallet_vane_xcm_transfer, pallet_vane_relay_asset, pallet_transfer_claim |
```
Comment (takahser) [inline applications/vane-network.md:269]: > Account who intend to extend vane product services must apply to be whitelisted to deploy the contract.

Do you refer to smart contracts here or to some kind of financial contract? If smart contract, does that mean each user has to deploy a smart contract in order to use vane?

File: applications/vane-network.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+| 3. | Notion template marketplace contract| The marketplace will showcase how we can enable transaction customization for social-commerce business and foster growth. It will contain, listing of notion templates, securely limiting access of the notion template, buying of the template, Order placement and tracking.|
```
Comment (takahser) [inline applications/vane-network.md:270]: Did I understand correctly - this is a marketplace to trade notion.so templates?
Could you please list the tech used here?

File: applications/vane-network.md [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+| 4. | Backend (Nest JS) | All chain ( vane, polkadot/rococo, assethub) api interaction implementation, API chart workflow, notion marketplace implementation for storing notion pages, notion users as a POC |
+| 5. | Frontend (Next JS) |  A front end prototype in Figma, A front end interaction for implemented feature hosted in vercel which will be only for wide screen ( no responsiveness ) as later on (not this grant ) we will implement a mobile application |
```
Comment (takahser) [inline applications/vane-network.md:272]:
```suggestion
| 5. | Frontend (Next JS) |  Front end wireframes in Figma and a NextJs-based front end for the implemented feature hosted in vercel which will be only for wide screen (no responsiveness) as later on (not this grant) we will implement a mobile application |
```

Comment (takahser) [reply applications/vane-network.md:272]: This would make it clearer that it's an actual implementation, not just figma wireframes.

File: applications/vane-network.md [hunk 8]
```diff
@@ -0,0 +1,317 @@
...
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Video tutorial | We will publish an workshop on X/Twitter that explains [...] (what was done/achieved as part of the grant and how the product should be used) |
+| 1. | Transaction Delivery Tracking System | As vane-risk free transfer system involves the following parts.<br> Relay_chain xcm reserveTransfer to Vane, confirmation of sender and receiver's address,sending xcm instructions to relay/assethub for transacting asset from vane para account to receiver.<br> The following are notable issues which may arrive; Failure to deliver and execute XCM message to vane, and relay/assethub.<br><br> A tracking functionality of where the transaction stuck is essential and will be implemented as a struct of an enum and stored in transaction tracking storage with key of multi_id.<br> The updating will be based on queries of effect of what the transaction is supposed to do using XCM.<br>|
```
Comment (takahser) [inline applications/vane-network.md:240]: Is this a pallet? Pls specify directly in the doc.

File: applications/vane-network.md [hunk 9]
```diff
@@ -0,0 +1,317 @@
...
+
+
+Vane is a parachain that act as an extension product and safety layer for Polkadot and AssetHub which provides services to help trust and growth on social commerce business and introducing transaction safety in web3.
+
+
+Vane-network consist of the following functionalities
+
+ - Confirmation and reversible transactions
+ - Customizable and trustless transactions for social-commerce business
+ - Native token-less parachain enabling usage of DOT, USDT & USDT for all interactions via XCM
```
Comment (takahser) [inline applications/vane-network.md:26]: Does this mean your solution enables non-native (e.g. USDT) transfers without the necessity to own the native asset? I believe this isn't included in the milestones yet, could you fix that?

**Review by takahser — CHANGES_REQUESTED** [review]
@MrishoLukamba thanks for adding the wireframes. I've did another iteration and still found some unclarities, please have a look.

File: applications/vane-network.md (hunk 5 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:269]: No user can extend vane functionalities by deploying a smart contract for an application that uses vane chain extensions and base layer functionality ( risk free and trustless transfers)

File: applications/vane-network.md (hunk 6 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:270]: Yes it is, and the tech stack used I listed above. TS, Nest Js, Next Js, Rust Ink Smart contract. ( showcasing extending vane functionality utilizing risk free transfers

File: applications/vane-network.md (hunk 7 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:272]: We did mention that it will figma wireframes and the actual front end

File: applications/vane-network.md (hunk 8 above)
Comment (MrishoLukamba) [reply applications/vane-network.md:240]: Yes it is a pallet

**Review by semuelle — COMMENTED** [review]
@MrishoLukamba, thanks for the updates. After reviewing the application again, I'm thinking it might be a good fit for the [Decentralized Futures Program](https://futures.web3.foundation/). Have you looked into that? If you are planning to build a fully-fledged product with business model, it's actually a much better fit.

**Comment by MrishoLukamba** [comment]
> @MrishoLukamba, thanks for the updates. After reviewing the application again, I'm thinking it might be a good fit for the [Decentralized Futures Program](https://futures.web3.foundation/). Have you looked into that? If you are planning to build a fully-fledged product with business model, it's actually a much better fit. 

I do believe for our stage web3 grants does make sense more.
We have our POC but we are still tinkering and finding optimal solutions to the problem and the future we want to create.

I do think decentralised futures will be the next step after this completion.

**Review by keeganquigley — APPROVED** [review]

**Comment by semuelle** [comment]
Thanks again for the application, @MrishoLukamba, and sorry for the long radio silence. The committee has discussed your proposal and decided not to go forward with it. The main reason being that it thinks the project to be a better fit for the DF program. So, we would like to recommend to apply there. Applications are only open until the end of June, you could always revisit the grants program as an alternative afterwards.
